src/vivarium_census_prl_synth_pop/constants/paths.py

```diff
@@ -15,6 +15,11 @@
 PERSONS_DATA_DIR = PROJECT_ROOT / "data/raw_data/current/United_States/"
 
 HOUSEHOLD_MOVE_RATE_PATH = BASE_DIR / "data/raw_data/move_rates_including_gq.csv"
+BUSINESS_NAMES_DATA = BASE_DIR / "data/raw_data/business_names.csv.bz2"
+# todo: where do we want to save this: share or repo?
+BUSINESS_NAMES_DATA_ARTIFACT_INPUT_PATH = (
```

Comment:
Just be sure if you decide the repo you make it accessible with `pip install` (no -e) by adding it to the MANIFEST (or package_data if this repo doesn't have a MANIFEST...I'm not familiar with where exactly those files come from) https://stackoverflow.com/questions/1612733/including-non-python-files-with-setup-py

Reply:
The will likely be useful here.  This is going to be a pretty long series (like at least 10 million names) so I have where creating it will save it but keeping it in the repo does not seem like a good idea.

Reply:
Consider putting that in the artifact.

Why would you want a non-editable install of a research repo under development?